### ISSUE TITLE
Suggestion: trigger roll20 macro on condition change

### ISSUE THREAD

**Issue by kakaroto** [kakaroto/Beyond20#103]
add option to trigger a macro for condition tracking which can then be used to set a custom status icon on tokens for example.

**Comment by ConnorJBeck** [comment]
This would be so goooood

**Comment by Aeristoka** [comment]
I concur, this would be an excellent feature. I spend a large amount of time assigning condition markers (or forgetting to) and having to be reminded by my players something is going on.

**Comment by ConnorJBeck** [comment]
Maybe a further enhancement, but it would be pretty dope if you cast a spell that requires concentration and could trigger assignment of a condition in roll20 to the token (to mark that the token is concentrating).

**Comment by kakaroto** [comment]
Yes! I never remember when my players are supposed to be concentrating!

**Comment by Aeristoka** [comment]
> Yes! I never remember when my players are supposed to be concentrating!

I try to get my players to mark the clock on themselves when they're concentrating, and the Ninja Mask when they're stealthed, and I still have baddies try to attack the Rogue who rolled a 29 on Stealth...

**Comment by kakaroto** [comment]
I started on this but it's a much more complex issue that I'm leaving for the next release before I can tackle it.
